### ISSUE TITLE
Use of OpenStreetMap (OSM) instead of Google Maps (Gmaps)

### ISSUE THREAD

**Issue by linux-lukas** [gematik/E-Rezept-App-Android#13]
TL;DR Please make sure that Gematik makes it at least once to deliver a good digital health app - take responsibility!

Personally, I would urge that the integration of Google Maps (GMaps; especially since without obtaining consent....) from the e-prescription app for Android be replaced AS FAST AS POSSIBLE(!) with a local integration of OpenStreetMap (OSM). 

From my point of view, in view of the "Telecommunications Telemedia Data Protection Act" (TTDSG) and the General Data Protection Regulation, there is no leeway whether, but it is essential to include a local and offline available integration of a digital map.
```[tasklist]
### Tasks
- [ ] Find Google Maps integration
- [ ] Remove Google Maps integration
- [ ] Implement local and offline OpenStreetMap integration
```

**Comment by pschneider1968** [comment]
WTF?! This app depends on ANY Google service?? Is that true? If so, I have only one question to the development team: "ARE YOU F*CKING CRAZY?"

You can't be serious about this. I think that the German Federal Data Protection Officer needs to be involved, if this turns out to be true.

**Comment by fnordlicht** [comment]
Hi @Lukas2112,

thank you very much for filing this issue. We'll look into it and post updates here. Stay tuned!

**Comment by dineshvg** [comment]
We would in the future release make changes that the user can decide if they want to use google maps or open street maps.

**Comment by Chris56** [comment]
Google is an absolute nogo!!!
Read the google smallprint (you will need a lot of time!) !!!
How can a publicly funded application use any google services ????

**Comment by Chris56** [comment]
Das Issue zu schließen mit einem "Ja vielleicht" und "vielleicht aber auch nicht" ist ein Unding. Es ist NICHT "completed" !!!
Entweder schließt man es mit: werden wir nicht implementieren und lassen die Fa. Google alle Stadortabfragen auswerten oder
man lässt es offen bis es implementiert ist.
